### PR TITLE
print a message after creating an environment

### PR DESCRIPTION
**Pull request by pradyunsg**
Resolves https://github.com/pypa/virtualenv/issues/1557

### CODE REVIEW

File: src/virtualenv/run/__init__.py [hunk 1]
```diff
@@ -20,6 +20,12 @@ def run_via_cli(args):
     """
     session = session_via_cli(args)
     session.run()
+
+    logging.warn(
+        "Created virtual environment via %s, using %s",
+        six.ensure_text(str(session.creator)),
+        six.ensure_text(str(session.seeder)),
+    )
```
Comment (pfmoore) [inline src/virtualenv/run/__init__.py]: Note that if #1568 is not accepted, the message here should also include the destination directory. (I don't really mind whether it does in the case where the user explicitly specifies the directory name, but it should if the default is used).

Comment (gaborbernat) [reply src/virtualenv/run/__init__.py]: the creator contains the destination directory 👍

Comment (pfmoore) [reply src/virtualenv/run/__init__.py]: Ah, that's great then. Sorry, I didn't have enough time to check the details.

Comment (pradyunsg) [reply src/virtualenv/run/__init__.py]: Yea, I spent quite a few minutes figuring that out myself. :)

Comment (gaborbernat) [reply src/virtualenv/run/__init__.py]: Sad to hear that 😢

**Comment by gaborbernat** [comment]
Given #1568 I'm tempted to mark this won't do now.

File: docs/cli_interface.rst [hunk 2]
```diff
@@ -10,8 +10,8 @@ CLI flags
 
 It modifies the environment variables in a shell to create an isolated Python environment, so you'll need to have a
 shell to run it. You can type in ``virtualenv`` (name of the application) followed by flags that control its
-behaviour. All options have sensible defaults, so you'll get a working virtual environment in a ``venv`` folder even
-if you don't pass any options. The default values for the command line options can be overridden via the
+behaviour. All options have sensible defaults, and there's on required argument: then name/path of the virtual
```
Comment (pradyunsg) [inline docs/cli_interface.rst]:
```suggestion
behaviour. All options have sensible defaults, and there's one required argument: then name/path of the virtual
```

File: docs/changelog/1548.feature.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Print out a one line message about the created virtual environment when no :option:`verbose` is set, this can now be
+sileneced to get back the original behaviour via the :optio=:`quiet` flag - by :user:`pradyunsg`.
```
Comment (nsoranzo) [inline docs/changelog/1548.feature.rst]:
```suggestion
silenced to get back the original behaviour via the :optio=:`quiet` flag - by :user:`pradyunsg`.
```

File: docs/changelog/1557.feature.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Print out a one line message about the created virtual environment when no :option:`verbose` is set, this can now be
+sileneced to get back the original behaviour via the :optio=:`quiet` flag - by :user:`pradyunsg`.
```
Comment (nsoranzo) [inline docs/changelog/1557.feature.rst]:
```suggestion
silenced to get back the original behaviour via the :optio=:`quiet` flag - by :user:`pradyunsg`.
```

File: docs/cli_interface.rst (hunk 2 above)
Comment (pfmoore) [reply docs/cli_interface.rst]: Also, "the name/path", not "then name/path"

**Pull request merged**